plugins/opener/src/scope_entry.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+use std::path::PathBuf;
+
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+pub enum Application {
```

Comment:
This needs to be untagged or the scope can't be parsed

```suggestion
#[serde(untagged)]
pub enum Application {
```